config/app_environment_variables.sample.rb

```diff
@@ -20,6 +20,7 @@
 # ENV['C_RATER_FAKE']                    ||= 'true'
 ENV['ROLLBARS_KEY']                      ||= 'XXXXXXX'
 ENV['LABBOOK_PROVIDER_URL']              ||= 'https://labbook.concord.org'
+ENV['ACTIVITY_PLAYER_URL']               ||= 'https://activity-player.concord.org/branch/master'
```

Comment:
We are trying to move away from using this file. So I'd remove this line to keep things more simple.